roles/common/templates/default.j2

```diff
@@ -3,7 +3,7 @@ DATABASE_NAME={{ database_name }}
 DATABASE_USER={{ database_user }}
 DATABASE_PASSWORD={{ database_password }}
 SECRET_TOKEN={{ secret_token }}
-MAIL_LINK_HOST=www.timeoverflow.org
+MAIL_LINK_HOST={{ mail_link_host }}
```

Comment:
We can use the Ansible variable `inventory_hostname` if we want to use the host name in every environment.